### PR TITLE
integrated climber

### LINKED ISSUE #10: Space Addition
**Describe the solution you'd like**
We need a space in `settings.gradle`

### LINKED ISSUE #18: Add Protobot Climber
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by AmnesiaExe**
**What this PR does**  
Climber
**What does it fix/add?**  
_Reference issues with "Fixes #10" or "Closes #10", where #10 is the number assigned to the issue_
Fixed #22

### CODE REVIEW

File: 4788/src/main/include/RobotMap.h [hunk 1]
```diff
@@ -115,8 +115,17 @@ struct RobotMap {
...
+
+    
+
   }; DrivebaseSystem drivebaseSystem;
 
+  struct ClimberSystem {
+    wml::actuators::DoubleSolenoid leftClimberSolenoid{ ControlMap::PCModule, wml::actuators::PneumaticsModuleType::kCTRE,ControlMap::leftClimberPort1, ControlMap::leftClimberPort2, 0.1};
+    wml::actuators::DoubleSolenoid rightClimberSolenoid{ ControlMap::PCModule, wml::actuators::PneumaticsModuleType::kCTRE,ControlMap::rightClimberPort1, ControlMap::rightClimberPort2, 0.1};
+  }; ClimberSystem climberSystem;
+};
```
Comment (CJBuchel) [inline 4788/src/main/include/RobotMap.h]: Extra bracket (code assumes end of robotmap) causes fail

File: 4788/src/main/include/ControlMap.h [hunk 2]
```diff
@@ -80,9 +83,8 @@ struct ControlMap {
   inline static const wml::controllers::tAxis drivebaseR{Driver, XboxController::kRightYAxis};
   
   // Climber
-   static constexpr int ClimberPort = 0;
-   static constexpr int ClimberSolenoidPort1 = 0;
-   static constexpr int ClimberSolenoidPort2 = 0;
+ inline static const wml::controllers::tButton ClimberToggle{ CoDriver, XboxController::kA };
```
Comment (CJBuchel) [inline 4788/src/main/include/ControlMap.h]: indentation not standard to the rest of file. (1 space less)

**Review by CJBuchel — CHANGES_REQUESTED** [review]
Fix RobotMap

File: 4788/src/main/include/RobotMap.h (hunk 1 above)
Comment (goanna247) [reply 4788/src/main/include/RobotMap.h]: Fixed

File: 4788/src/main/include/ControlMap.h (hunk 2 above)
Comment (goanna247) [reply 4788/src/main/include/ControlMap.h]: Fixed

**Review by CJBuchel — APPROVED** [review]

**Pull request merged**